### ISSUE TITLE
[UX] Login settings: Consolidate all "Limit login attempts" settings.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#6567]
The form in `admin/config/people/login` has the following groups of settings:
- User login (username, email, or both)
- Limit login attempts by IP address
- Limit login attempts by user
- Log excessive failed login attempts ("roque", free-floating checkbox)

I propose that we group the 3 last items together, under a single "Failed login attempts settings" fieldset, which should be collapsed by default (since these settings are kinda advanced, and the defaults should just work for most cases).

Having the fieldset collapsed by default should also leave more room in this form for the settings that we are looking to introduce as part of #3309, which are less advanced.

We should also remove instances of "user" from labels and help text. There's quite a few of them in this form.

**Comment by klonos** [comment]
PR ready for initial UI review (tests still running): https://github.com/backdrop/backdrop/pull/4763

Before/after screenshot (the new fieldset is collapsed by default - expanded here):
![image](https://github.com/backdrop/backdrop-issues/assets/2423362/7c45b44f-f83e-4e63-8e97-7bbe30f90235)
...and how the form looks by default, with the fieldset collapsed:
![image](https://github.com/backdrop/backdrop-issues/assets/2423362/8fa6ce76-d301-4ff1-a510-15d4d688a1a8)

There are many text changes, because:
- there were several instances of "user" instead of "user account"
- text that would apply to multiple elements was consolidated and moved to the main fieldset instead
- help text that was stating the obvious or simply repeating the same text as in the respective field labels was removed
- the period that was previously added to the `Log excessive failed login attempts.` checkbox label needed to be removed (we are not adding periods to labels)

**Comment by kiamlaluno** [comment]
I like that the *If there are excessive failed login attempts, the offending IP address or user account will be temporarily blocked.* sentence is placed at the before the relative settings, and not at the bottom of the form as it is now. I would not expect that the description for the *Log excessive login attempts* checkbox would also explain what all those settings are (and that is probably the reason I have never read that description).

**Comment by klonos** [comment]
I was also thinking to re-word the help text for the two radio options.

Current:
- More secure; more likely to block user accounts.
- Less secure; less likely to block user accounts.

Considering this instead (bolding the bits that are important):
- More secure; but **more likely to block user accounts**.
- Less likely to block user accounts, **but less secure**.

And some visuals:
![image](https://github.com/backdrop/backdrop-issues/assets/2423362/43c7557c-8f50-4500-b143-e08e6ddbf0d2)

...basically emphasizing the bits that would otherwise be a warning for caution.

**Comment by klonos** [comment]
...now that I'm thinking about it, I liked how the original versions of these texts mentioned the words "lock out" instead of "block". I had to change it to "block" because I changed "users" to "user accounts", but if we changed that to "people" instead, it could be "lock people out" (of the site) instead of "block user accounts". I think I will change the PR to that instead.

**Comment by klonos** [comment]
...PR updated. Here's before/after of that specific part of the form:
![image](https://github.com/backdrop/backdrop-issues/assets/2423362/8adf24fd-d5a8-4891-a6d1-3463e0a65131)

**Comment by klonos** [comment]
...I also don't really like `less secure`, especially considering the fact that we have that as the default. Perhaps say `less strict` or `less accurate` instead? Thoughts?

**Comment by kiamlaluno** [comment]
I would rather not use _secure_ nor _strict_. _More likely to lock people out of the site_ and _Less likely to lock people out of the site_ should be sufficient.

**Comment by klonos** [comment]
Thanks for the feedback @izmeez and @kiamlaluno 🙏🏼 

What do you thing about using `more accurate`/`less accurate`? ...if these don't sound good, then I'm also inclined to go with @kiamlaluno's suggestion and just omit the `more secure`/`less secure` altogether. ...would love some feedback from others on that.

**Comment by kiamlaluno** [comment]
To make my previous comment clearer: I understand that *secure* means *avoiding that people who try to guess the password used for an account to successfully do so*. I would just avoid to define a method more secure than the other, and just say that a method could lock out people who try to access their own account.

**Comment by klonos** [comment]
Perhaps, we should just say exactly that(?):
> Makes guessing passwords harder, but is more likely to lock people out of the site.

...I don't think that we should say the opposite ("makes guessing passwords easy") for the other option though. Perhaps throwing the word "even" into the mix would improve things:
> Makes guessing **even** passwords harder, but is more likely to lock people out of the site.

**Comment by klonos** [comment]
...although "makes guessing passwords harder" is not a very accurate description of what this setting does from a technical point of view 🤔 ...it doesn't make guessing harder per se - it blocks more guessing attempts.

**Comment by kiamlaluno** [comment]
That's correct: It just limits the number of wrong passwords can be entered.
As for *Makes guessing passwords harder*, that is not necessary true: In the case I know the password associated with an account is the person's date birthday, neither one or the other method makes guessing the password harder for me.

**Comment by klonos** [comment]
Thanks for all the feedback @kiamlaluno 🙏🏼 ...it is really helpful. I think I'll wait for another person or two to chime in, and then will most likely remove the mentions of "more secure"/"less secure" entirely (unless people think that they do actually have value and are accurate descriptions, or can suggest any better alternative).

**Comment by klonos** [comment]
...one final thought with regards to that setting specifically: should we move the default option (UID + IP combo) to be the first of the two available options? It feels odd the way it is placed last now.
